Library/Formula/nccmp.rb

```diff
@@ -0,0 +1,21 @@
+class Nccmp < Formula
+  desc "nccmp compares 2 NetCDF files bitwise, semantically or with a tolerance."
```

Comment:
The name will automatically prepend the `desc` output, so you don't need to start with `nccmp` here.